apps/src/templates/HeadingBanner.jsx

```diff
@@ -5,39 +5,77 @@
 
 import React from 'react';
 import color from "../util/color";
+import ProgressButton from './progress/ProgressButton';
+import i18n from "@cdo/locale";
 
 const styles = {
   bannerHeading: {
     fontFamily: '"Gotham 7r", sans-serif',
     color: color.white,
     fontSize: 32,
-    marginTop: 40,
     marginBottom: 10,
     lineHeight: '40px'
   },
   bannerSubHeading: {
     fontFamily: '"Gotham 4r", sans-serif',
     color: color.white,
-    fontSize: 13,
-    marginBottom: 145
+    fontSize: 16,
+    lineHeight: '32px',
+    marginBottom: 10
+  },
+  bannerDescription: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.white,
+    fontSize: 16,
+    width: '600px',
+    marginBottom: 20,
+  },
+  spacer: {
+    marginBottom: '50px'
+  },
+  bigSpacer: {
+    marginBottom: '150px'
   }
 };
 
 const HeadingBanner = React.createClass({
   propTypes: {
+    extended: React.PropTypes.bool,
     headingText: React.PropTypes.string,
-    subHeadingText: React.PropTypes.string
+    subHeadingText: React.PropTypes.string,
+    description: React.PropTypes.string,
   },
 
   render() {
+    const {extended, headingText, subHeadingText, description} = this.props;
+
     return (
       <div>
+        {!extended && (
+          <div style={styles.spacer}/>
+        )}
         <div style={styles.bannerHeading}>
-          {this.props.headingText || <span>&nbsp;</span>}
+          {headingText || <span>&nbsp;</span>}
         </div>
         <div style={styles.bannerSubHeading}>
-          {this.props.subHeadingText || <span>&nbsp;</span>}
+          {subHeadingText || <span>&nbsp;</span>}
         </div>
+        {extended ? (
```

Comment:
`extended` seems more specific than just an extension, and seems to explicitly be a sign-up UI, so I'd probably rename it to `showSignup` or something like that to speak to that intended purpose.  

(Alternatively we could pass in a child component for the extended section, but given we only have one kind of extended component, I'd hold off on that until we refactor to support more than one type of extension.)